orchagent/routeorch.cpp

```diff
@@ -1874,6 +1880,25 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
         }
         else
         {
+            if (!blackhole && vrf_id == gVirtualRouterId && ipPrefix.isDefaultRoute())
```

Comment:
Does this mean we set packet action for the default route every time even if it is not changed? Is there any possibility to set it only when required?

Reply:
Setting it to DROP was done in removeRoute and setting it to FORWARD is in addRoute and there is no global state recording this.